### DISCUSSION TITLE
download limits

### DISCUSSION THREAD

**Discussion by dondim0n** [nextcloud/all-in-one#5199] [Questions] [answered]
Task:  
Limit the number of files that can be downloaded by a single user within a given period (e.g., a day). From a specific directory or the whole system.
The download limit should be customizable depending on the type of user (regular user, manager, administrator), file size or other parameters.
Keep a log of all downloads with user, time, IP address, and download information.
Notify the user when the download limit is approached.
Blocking the ability to download and displaying a warning message when the limit is exceeded.
Notifying the administrator when a user reaches the download limit.

Does anyone have similar experience?

**Comment by szaimen** [comment, marked as answer]
Hi, feel free to open a feature request for this here: https://github.com/nextcloud/server/issues
